cloud/linode/loadbalancers.go

```diff
@@ -187,7 +187,7 @@ func (l *loadbalancers) cleanupOldNodeBalancer(ctx context.Context, service *v1.
 // GetLoadBalancerName returns the name of the load balancer.
 //
 // GetLoadBalancer will not modify service.
-func (l *loadbalancers) GetLoadBalancerName(ctx context.Context, clusterName string, service *v1.Service) string {
+func (l *loadbalancers) GetLoadBalancerName(_ context.Context, _ string, _ *v1.Service) string {
 	unixNano := strconv.FormatInt(time.Now().UnixNano(), 16)
```

Comment:
I think we should start adding in the clusterName here to the nb name. thoughts?

Reply:
can discuss further https://github.com/linode/linode-cloud-controller-manager/issues/150

merging this PR since it's kinda outta conext for fixing the linter